index.html

```diff
@@ -1,37 +1,192 @@
+                <p class="text2">Back End Dev</p>
+                <img class="dot" src="Counter.png" alt="counter">
+                <p class="text2">2015</p>
+            </section>
+            <p class="cardText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <section class="cards">
+                <p class="firstcard">HTML</p>
+                <p class="firstcard">CSS</p>
+                <p class="firstcard">javascript</p>
+            </section>
+            <section>
+                <p class="buttonProject">See Project</p>
+            </section>
+        </section>
+        <section class="content4">
+            <img class="snapshot" src="SnapshootPortfolio(5).png" alt="picture">
+            <h1 class="text">Multi-Post <br> Stories</h1>
+            <section class="block">
+                <p class="text1">CANOPY</p>
+                <img class="dot" src="Counter.png" alt="counter">
+                <p class="text2">Back End Dev</p>
+                <img class="dot" src="Counter.png" alt="counter">
+                <p class="text2">2015</p>
+            </section>
+            <p class="cardText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <section class="cards">
+                <p class="firstcard">HTML</p>
+                <p class="firstcard">CSS</p>
+                <p class="firstcard">javascript</p>
+            </section>
+            <section>
+                <p class="buttonProject">See Project</p>
+            </section>
+        </section>
+    </section>
+
+
+
+    <section class="about">
+        <h1 class="nameAbout">About<br>Myself</h1>
+        <p class="introAbout">Hello I'm a software developer! I can help you build a product , feature or website Look
+            through some of my work and experience! If you like what you see and have a project you need coded, don't
+            hestiate to contact me.</p>
+        <p class="touch">LET'S CONNECT</p>
+        <section class="icon">
+            <ul class="aboutList">
+                <li><a href="https://twitter.com/NeoNabil2"><img class="icons" src="twitter.png"
+                            alt="socialmediaicon"></a>
+                </li>
+                <li><a href="https://www.linkedin.com/"><img class="icons" src="Linkedinicon.png"
+                            alt="socialmediaicon"></a>
+                </li>
+                <li><a href="https://medium.com/"><img class="icons" src="medium.png" alt="socialmediaicon"></a></li>
+                <li><a href="https://github.com/NabilHY"><img class="icons" src="github.png" alt="socialmediaicon"></a>
+                </li>
+                <li><a href="https://angel.co/"><img class="icons" src="angellist.png" alt="socialmediaicon"></a></li>
+            </ul>
+        </section>
+        <section>
+            <p class="button1">Get My resume</p>
+        </section>
```

Comment:
- [ ] You wouldn't need to do use a `section` tag here, because this `button` alone is not significant enough to be an entire `section`.
- [ ] Please avoid using another tag for something that should ideally be an `a` or `button` tag. here you can simply use a `button` tag.
```suggestion
        <div>
            <button class="button1">Get My resume</button>
        </div>
```